docs/_Sidebar.md

```diff
@@ -0,0 +1,29 @@
+* [Home](Home)  
+* [Issues](Issues)
+* [Implementation in Arduino IDE](Implementation-in-Arduino-IDE)
+  * [Linux 64 bit Flasher to board](Linux-64-bit-Flasher-to-board "Getting Segger and XMC-Flasher to work on 64 bit Linux")
+  * [Windows 10 64 bit Flasher](Windows-10-64-bit-Flasher "Getting Segger and XMC-Flasher to work on 64 bit Windows 10")
+
+* [Built-in Libraries](Built-in-Libraries)
+* Supported Boards
+  * [XMC1100 2Go](XMC1100-2Go)
+  * [XMC1100 Boot Kit](XMC1100-Boot-Kit)
+  * [XMC1400 2Go](XMC1400-2Go)
+  * [XMC1400 Kit for Arduino](XMC1400-Kit-for-Arduino)
+  * [XMC4200 Platform2Go](XMC4200-Platform2Go)
+  * [XMC4400 Platform2Go](XMC4400-Platform2Go)
+  * [XMC4700 Relax Kit](XMC4700-Relax-Kit)
+* Legacy board (Supported until 1.7.0)
+  * [XMC1100 H-Bridge 2Go](XMC1100-H-Bridge-2Go)
+  * [XMC1300 Boot Kit](XMC1300-Boot-Kit)

```

Comment:
as mentioned in the other comment. The XMC1300 Boot Kit is not a legacy board but a supported board.

Reply:
I will change it :D 
A question: https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1300-Boot-Kit the link in this page has been deprecated.  Any Update needed?

Reply:
Good point. I believe since the docs are to be migrated to RTD, they have to be redone anyway. So I think this is fine for now. @jaenrig-ifx can further comment.